pkg/controllermanager/controller/shoot/shoot_retry_control.go

```diff
@@ -121,11 +122,10 @@ func isShootFailed(shoot *gardencorev1beta1.Shoot) bool {
 		shoot.Generation == shoot.Status.ObservedGeneration
 }
 
-func mustRetryNow(shoot *gardencorev1beta1.Shoot, retryPeriod metav1.Duration) (bool, time.Duration) {
+func mustRetryNow(shoot *gardencorev1beta1.Shoot, retryPeriod metav1.Duration, syncJitterPeriod *metav1.Duration) (bool, time.Duration) {
```

Comment:
You can consider the following change:
```diff
--- a/pkg/controllermanager/controller/shoot/shoot_retry_control.go
+++ b/pkg/controllermanager/controller/shoot/shoot_retry_control.go
@@ -94,7 +94,7 @@ func (r *shootRetryReconciler) Reconcile(ctx context.Context, request reconcile.

        mustRetry, requeueAfter := mustRetryNow(shoot, *r.config.RetryPeriod, r.config.RetryJitterPeriod)
        if !mustRetry {
-               shootLogger.Infof("[SHOOT RETRY] Scheduled retry in %s", requeueAfter.Round(time.Minute))
+               shootLogger.Infof("[SHOOT RETRY] Scheduled retry in %s", requeueAfter.Round(time.Second))
                return reconcile.Result{RequeueAfter: requeueAfter}, nil
        }
```

As now we have random jitterPeriod we can be more precise when logging the `requeueAfter`. Otherwise values like `14m18s` or `13m46s` will be all logged as `14m0s`.